pkg/editors/lookup_darwin.go

```diff
@@ -0,0 +1,251 @@
+	Name string
+
+	// List of bundle identifiers that are used by the app in its multiple versions.
+	BundleIdentifiers []string
+}
+
+// This list contains all the external editors supported on macOS. Add a new
+// entry here to add support for your favorite editor.
+var editors = []DarwinExternalEditor{
+	{
+		Name:              "Atom",
+		BundleIdentifiers: []string{"com.github.atom"},
+	},
+	{
+		Name:              "Aptana Studio",
+		BundleIdentifiers: []string{"aptana.studio"},
+	},
+	{
+		Name:              "MacVim",
+		BundleIdentifiers: []string{"org.vim.MacVim"},
+	},
+	{
+		Name:              "Neovide",
+		BundleIdentifiers: []string{"com.neovide.neovide"},
+	},
+	{
+		Name:              "VimR",
+		BundleIdentifiers: []string{"com.qvacua.VimR"},
+	},
+	{
+		Name:              "Visual Studio Code",
+		BundleIdentifiers: []string{"com.microsoft.VSCode"},
+	},
+	{
+		Name:              "Visual Studio Code (Insiders)",
+		BundleIdentifiers: []string{"com.microsoft.VSCodeInsiders"},
+	},
+	{
+		Name:              "VSCodium",
+		BundleIdentifiers: []string{"com.visualstudio.code.oss", "com.vscodium"},
+	},
+	{
+		Name: "Sublime Text",
+		BundleIdentifiers: []string{
+			"com.sublimetext.4",
+			"com.sublimetext.3",
+			"com.sublimetext.2",
+		},
+	},
+	{
+		Name:              "BBEdit",
+		BundleIdentifiers: []string{"com.barebones.bbedit"},
+	},
+	{
+		Name:              "PhpStorm",
+		BundleIdentifiers: []string{"com.jetbrains.PhpStorm"},
+	},
+	{
+		Name:              "PyCharm",
+		BundleIdentifiers: []string{"com.jetbrains.PyCharm"},
+	},
+	{
+		Name:              "PyCharm Community Edition",
+		BundleIdentifiers: []string{"com.jetbrains.pycharm.ce"},
+	},
+	{
+		Name:              "DataSpell",
+		BundleIdentifiers: []string{"com.jetbrains.DataSpell"},
+	},
+	{
+		Name:              "RubyMine",
+		BundleIdentifiers: []string{"com.jetbrains.RubyMine"},
+	},
+	{
+		Name:              "RStudio",
+		BundleIdentifiers: []string{"org.rstudio.RStudio", "com.rstudio.desktop"},
+	},
+	{
+		Name:              "TextMate",
+		BundleIdentifiers: []string{"com.macromates.TextMate"},
+	},
+	{
+		Name:              "Brackets",
+		BundleIdentifiers: []string{"io.brackets.appshell"},
+	},
+	{
+		Name:              "WebStorm",
+		BundleIdentifiers: []string{"com.jetbrains.WebStorm"},
+	},
+	{
+		Name:              "CLion",
+		BundleIdentifiers: []string{"com.jetbrains.CLion"},
+	},
+	{
+		Name:              "Typora",
+		BundleIdentifiers: []string{"abnerworks.Typora"},
+	},
+	{
+		Name:              "CodeRunner",
+		BundleIdentifiers: []string{"com.krill.CodeRunner"},
+	},
+	{
+		Name: "SlickEdit",
+		BundleIdentifiers: []string{
+			"com.slickedit.SlickEditPro2018",
+			"com.slickedit.SlickEditPro2017",
+			"com.slickedit.SlickEditPro2016",
+			"com.slickedit.SlickEditPro2015",
+		},
+	},
+	{
+		Name:              "IntelliJ",
+		BundleIdentifiers: []string{"com.jetbrains.intellij"},
+	},
+	{
+		Name:              "IntelliJ Community Edition",
```

Comment:
Maybe nicer to shorten this (and others) to `Intellij CE`?

Reply:
If you prefer? I don't think the user experience should suffer because we want to align the code base, so if we both agree the shorter name is better we should do something about it :)